host_core/lib/host_core/lattice/lattice_supervisor.ex

```diff
@@ -0,0 +1,125 @@
+defmodule HostCore.Lattice.LatticeSupervisor do
+  @moduledoc """
+  The lattice supervisor is responsible for starting the following children:
+  * A gnat connection supervisor for: control and rpc connections
+  * A consumer supervisor for the control interface subscription(s)
+  """
+
+  use Supervisor
+
+  require Logger
+
+  @spec start_link(HostCore.Vhost.Configuration.t()) :: :ignore | {:error, any} | {:ok, pid}
+  def start_link(config) do
+    Supervisor.start_link(__MODULE__, config,
+      name: HostCore.Lattice.LatticeRoot.via_tuple(config.lattice_prefix)
+    )
+  end
+
+  # Uses the same strongly typed host configuration as other supervisors in the hierarchy (though it only cares about the lattice info)
+  @impl true
+  def init(config) do
+    Logger.metadata(lattice_prefix: config.lattice_prefix)
+
+    Logger.info("Starting lattice supervisor for '#{config.lattice_prefix}'")
+
+    children =
+      [
+        Supervisor.child_spec(
+          {Gnat.ConnectionSupervisor, HostCore.Nats.control_connection_settings(config)},
+          id: HostCore.Nats.control_connection(config.lattice_prefix)
+        ),
+        Supervisor.child_spec(
+          {Gnat.ConnectionSupervisor, HostCore.Nats.rpc_connection_settings(config)},
+          id: HostCore.Nats.rpc_connection(config.lattice_prefix)
+        ),
+        Supervisor.child_spec(
+          {Gnat.ConsumerSupervisor,
+           %{
+             connection_name: HostCore.Nats.control_connection(config.lattice_prefix),
+             module: HostCore.ControlInterface.LatticeServer,
```

Comment:
I'm sure I'm missing it, but where are the subcriptions for things like launch actor/provider, stop/scale, update, etc?